src/rt-app_types.h

```diff
@@ -151,6 +151,7 @@ typedef struct _phase_data_t {
 	int loop;
 	event_data_t *events;
 	int nbevents;
+	int sched_prio;
```

Comment:
What about we embed a sched_param instead, and use the sched_prio field only for the moment?
So that we don't have to change it again for DEADLINE support.